scripts/run-with-strimzi.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+set -eo pipefail
+
+if [[ -z $QUAY_ORG ]]; then
+  echo "Please set QUAY_ORG, exiting"
+  exit 1
+fi
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+. "${SCRIPT_DIR}/common.sh"
+cd "${SCRIPT_DIR}/.."
+
+if [[ "${QUAY_ORG}" != 'kroxylicious' ]]; then
+  echo "building and pushing image to quay.io"
+  PUSH_IMAGE=y "${SCRIPT_DIR}/deploy-image.sh"
+else
+  echo "QUAY_ORG is kroxylicious, not building/deploying image"
+fi
+${MINIKUBE} start --memory=4096
+${KUBECTL} create namespace kafka || true
+${KUBECTL} create -f 'https://strimzi.io/install/latest?namespace=kafka' -n kafka
+${KUBECTL} apply -f install/kafka-persistent.yaml -n kafka
+${KUBECTL} wait kafka/my-cluster --for=condition=Ready --timeout=300s -n kafka 
+${SED} -i "s/quay\.io\/kroxylicious/quay\.io\/${QUAY_ORG}/g" install/*
+${KUBECTL} apply -f install -n kafka
+${KUBECTL} wait deployment/kroxylicious-proxy --for=condition=Available=True --timeout=300s -n kafka 
+
+echo "To produce to kroxylicious:"
+echo "kubectl -n kafka run kafka-producer -ti --image=quay.io/strimzi/kafka:0.35.0-kafka-3.4.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server kroxylicious-service:9292 --topic my-topic"
```

Comment:
Personally I think the pod name should be `kroxylicious-producer` but that possibly relates more to how I've been using the commands... Maybe it does make more sense to call it a `kafka-producer` for users who are not developing with kroxylcious.